star_tides/api/controllers/auth_controller.py

```diff
@@ -7,9 +7,13 @@
 class LoginController(Controller):
     ''' Controller to handle basic auth login (username and password)
     '''
-    def process_request(self):
+    def process_request(self) -> dict:
```

Comment:
I'm going to write an issue for this. I think that that could be very handy but I'm not sure of the best way to implement the serialization of the class because it's already a serialized object.